modules/tinymce/src/core/main/ts/keyboard/KeyboardOverrides.ts

```diff
@@ -31,6 +31,8 @@ const registerKeyboardOverrides = (editor: Editor) => {
   HomeEndKeys.setup(editor, caret);
   PageUpDownKeys.setup(editor, caret);
 
+  editor.addShortcut('Meta+P', '', 'mcePrint'); // TODO: check with RTC
```

Comment:
@TheSpyder @spocke will this work for RTC? Asking as this logic is disabled atm for RTC.

Reply:
We didn't enable the print plugin because we wanted to decide whether to print from the model - this will work, but it only prints the view.

It's probably fine.

Reply:
Except this won't work at all as long as RTC is enabled (or the shortcut at least) because it won't ever be registered. See https://github.com/tinymce/tinymce/blob/feature/TINY-8314/modules/tinymce/src/core/main/ts/keyboard/KeyboardOverrides.ts#L40-L44

Reply:
Oh right. Sorry, I misread your message. I thought the plugin was disabled for RTC. It's enabled, so this is a breaking change.

Reply:
I think I accidentally resolved this? Sorry if so. Is this going to be a problem then? Do I need to change anything in this PR?

Reply:
Yep. This shortcut needs to be registered even in RTC mode (I guess the `register` function needs to be inlined into `setup`, and the print registration moved to before the `if rtc` check).

Reply:
Have commited something, let me know if you want it changed

Reply:
looks good 👍